### DISCUSSION TITLE
Helm chart development

### DISCUSSION THREAD

**Discussion by shanisma** [juspay/hyperswitch#3218] [Ideas / Feature Requests]
I suggest 5 helm charts standalone apps:
- sdk (assets)
- Vault 
- App / server
- admin panel
- sdk demo 

Another umbrella chart to deploy end 2 end that will wrap SDK + App + Vault + Admin Panel ?

**Comment by shakthi-22** [comment]
Hey @shanisma Thanks for your input on enhancing our helm charts. Tracking this [here](https://github.com/juspay/hyperswitch-helm/issues/26) 
Looking forward to your contributions!

**Comment by shakthi-22** [comment]
We now have 3 helm charts for individual components as follows
- App Server + Control Center (Control center is tightly coupled with our app server as it uses the same APIs for fetching data and only has an additional UI component for ease of use)
- SDK (assets + demo app)
- Vault (for storing sensitive payment method info)

We also have an umbrella chart to deploy all the 3 individual helm charts together.

This will allow seamless and flexible deployments of individual components and enable selective rollout of upgrades, as required, to specific components.
